src/nvme/mi.c

```diff
@@ -414,10 +414,21 @@ static int nvme_mi_verify_resp_mic(struct nvme_mi_resp *resp)
 	return resp->mic != ~crc;
 }
 
+__attribute__((weak)) void *nvme_mi_submit_entry(__u8 type, const void *hdr, size_t hdr_len,
+						 const void *data, size_t data_len) { return NULL; }
+
+__attribute__((weak)) void nvme_mi_submit_exit(__u8 type, const void *hdr, size_t hdr_len,
```

Comment:
`struct nvme_mi_msg_resp` for `hdr` here too?

Reply:
This also fixed. Thank you.